### ISSUE TITLE
How to train the facenet model

### ISSUE THREAD

**Issue by Ram-Godavarthi** [shubham0204/FaceRecognition_With_FaceNet_Android#24]
Hi,

Could you please let me know how to train the facenet model ??
Any reference or links??

Thanks in advance

**Comment by shubham0204** [comment]
You can obtain a pretrained Keras model for FaceNet, from [DeepFace](https://github.com/serengil/deepface). Then load this model using `tf.keras.models.load_model()` and you're ready to train the model on your dataset.

**Comment by Ram-Godavarthi** [comment]
I saw the repo and but there is explanation about the dataset requirements and labeling stuff. 
Do you have python script for the training task ??
Any info about the training dataset and its labels required.??
